File: input_program/program_6.java
import java.util.Scanner;

public class MultiplicationTable {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter an integer to see its multiplication table: ");
        int N = scanner.nextInt();

        System.out.println("--- Table of " + N + " ---");
        for (int i = 1; i <= 10; i++) {
            System.out.println(N + " x " + i + " = " + (N * i));
        }
        scanner.close();
    }
}

File: translated/gemini/program_6.py
def main():
    N = int(input("Enter an integer to see its multiplication table: "))

    print("--- Table of " + str(N) + " ---")
    for i in range(1, 11):
        print(str(N) + " x " + str(i) + " = " + str(N * i))

if __name__ == "__main__":
    main()